pkg/controller/releasemanager/releasemanager_controller.go

```diff
@@ -605,5 +610,15 @@ func (r *ResourceSyncer) syncVirtualService() error {
 	}
 
 	r.log.Info("VirtualService sync'd", "Type", "VirtualService", "Audit", true, "Content", vs, "Op", op)
+	for _, incarnation := range r.incarnations.sorted() {
+		revisionReleaseWeightGauge.
+			With(prometheus.Labels{
+				"app":     appName,
+				"tag":     incarnation.tag,
+				"cluster": r.cluster.Name,
```

Comment:
The "cluster" label is automatically added by Thanos when aggregating the stats, maybe not needed?

Reply:
it's the target cluster... should I name it something else?

Reply:
I think to be on the safe side it might be best to rename to targetCluster or something.